app/models/concerns/patient_reportable.rb

```diff
@@ -35,8 +43,9 @@ module PatientReportable
       end
     end
 
-    def self.ltfu_time_ago(date)
-      (date.to_date - LTFU_TIME).end_of_month
+    # Returns a plain timestamp to be interpreted in reporting time, for use with materialized views
```

Comment:
What is a "plain timestamp" ? Can we expand on this comment for future timezone struggles?

```
Our legacy materialized views store their timestamps in _local time_, so IST for India for example. Because of this we need to adjust our application provided timestamps by the UTC offset, otherwise Rails automatic timezone conversion do the wrong thing, as it assumes times from the database are all UTC.
```

After writing all that out, this would really be much more clear if we always let Postgres convert timezones from the legacy mat views to UTC, because then Rails would do the right thing.  That would probably slow down all these queries though, and require more careful SQL.